packages/edit-site/src/components/global-styles/root-menu.js

```diff
@@ -73,6 +75,15 @@ function RootMenu() {
 						{ __( 'Layout' ) }
 					</NavigationButtonAsItem>
 				) }
+				{ hasBackgroundPanel && (
```

Comment:
Let's place this one at the top instead of bottom

Reply:
Moved it in [7f4951d](https://github.com/WordPress/gutenberg/pull/63216/commits/7f4951d474b52a0774532ee43a0d396f92352013)